python3/ltk/actions/init_action.py

```diff
@@ -55,6 +55,8 @@ def init_action(self, host, access_token, client_id, project_path, folder_name,
                             login_host = 'https://sso.lingotek.com'
                         elif 'clone.lingotek.com' in host:
                             login_host = 'https://clonesso.lingotek.com'
+                        elif 'int.lingotek.com' in host:
+                            login_host = 'https://intsso.lingotek.com'
```

Comment:
Does this need to be present in production? Or is this just necessary for testing on our local machine?

Reply:
Technically this and clone do not need to be here but you still have to make more changes to actually point to a different server. We could remove both and it would probably look cleaner